ProboCI/probo
# Feature/lol/drupal 8 settings

### Original Issue via @tizzo 
> Without an intermediary cache rebuild step we get intermittent build fails. On any Drupal 8 site we need to immediately rebuild the cache after the site is bootstrappable before we do anything else (including db updates, etc). I wonder if we should require the version to be specified - the other choice would be to include some conditional logic that we always append into the script and do the switching by asking drush the version after the code is downloaded and either built or symlinked.

### Solution
 - Added `drupalVersion` support to .probo.yaml file.
 - Default `drupalVersion` to 7.
 - If building a D8 site, clear caches immediately after the DB import.

### To test
 - Create a new branch for a Probo enabled project (needs to be one with a database that will import).
 - Add `drupalVersion: 8` to the .probo.yaml file (ok even if you don't have a D8 site for now).
 - Create a PR.
 - Check to see that the caches were cleared twice, once after the DB import and once at the end.
